message_ix/report/__init__.py

```diff
@@ -141,6 +141,7 @@
         "message::costs",
         "message::emissions",
     ),
+    ("message::sankey", "concat", "out::pyam", "in::pyam"),
```

Comment:
import logging
from functools import partial
from typing import Tuple, Mapping, List

from genno.operator import broadcast_map
from ixmp.report import Key
from .pyam import collapse_message_cols

# Assuming TASKS1 was where Sankey tasks were defined:
TASKS1 = (
    ("message::sankey", "concat", "out::pyam", "in::pyam"),
)

def get_sankey_tasks() -> List[Tuple[Tuple, Mapping]]:
    """Return a list of tasks for Sankey diagram reporting."""
    to_add: List[Tuple[Tuple, Mapping]] = []
    strict = dict(strict=True)

    # This might include specific Sankey diagram configuration or additional tasks.
    for t in TASKS1:
        to_add.append((t, strict))

    return to_add

class SankeyReporter:
    """A specialized reporter for generating Sankey diagrams."""

    @staticmethod
    def add_tasks(reporter, fail_action: str = "raise") -> None:
        """Add Sankey-related tasks to a given reporter."""
        reporter.add_queue(get_sankey_tasks(), fail=fail_action)

```suggestion
    ("message::sankey", "concat", "out::pyam", "in::pyam"),
```

Reply:
Thanks for this suggestion, @daymontas1 :)
To clarify here: are these the contents of the new `sankey.py` file you mentioned above? They probably don't need to include the `suggestion` part, then. 
If I understand correctly, what you want to suggest is deleting this line that is no longer needed here:
```suggestion

```